xray/analyze.py

```diff
@@ -22,7 +22,8 @@ def __init__(self, csv_path):
             reader = csv.reader(csvfile, delimiter=",", quotechar="|")
             next(reader)
             for row in reader:
-                self.indices.append(int(row[0]))
+                if len(row[0]) > 0:
```

Comment:
When the previous code was added a receive index of 0 meant that the packet hadn't arrived, but it's better represented with empty data. After making that change, this part of the code wasn't updated to handle empty index fields which would cause exceptions when some packet(s) didn't arrive